scripts/update-sdk.sh

```diff
@@ -17,9 +17,12 @@
 set -euo pipefail
 
 LATEST_SDK_TAG=$(curl -sSfL -X GET  https://api.github.com/repos/mongodb/atlas-sdk-go/releases/latest | jq -r '.tag_name')
-
 LATEST_SDK_RELEASE=$(echo "${LATEST_SDK_TAG}" | cut -d '.' -f 1)
-echo  "==> Updating SDK to latest major version ${LATEST_SDK_TAG}"
+
+echo  "==> Updating SDK to latest major version tag: ${LATEST_SDK_TAG}, release: ${LATEST_SDK_RELEASE}"
+curl -sSfL -X GET  "https://proxy.golang.org/go.mongodb.org/atlas-sdk/${LATEST_SDK_RELEASE}/@v/${LATEST_SDK_TAG}.info"
```

Comment:
FYI this may trigger the go proxy to do a fetch but it happens in the background so the next step may still fail